### ISSUE TITLE
Supplement WPILib's inability to run Kotlin code in simulation from VSCode

### ISSUE THREAD

**Issue by BrenekH** [BrenekH/kotlin-for-frc#63]
When attempting to use VSCode-WPILib's simulate command, it fails because the Java Debugger VSCode extension does not play nicely with Kotlin code.

Kotlin for FRC can alleviate this by providing its own command that doesn't do any debugging, but runs the `simulateJava` Gradle task so that the simulation at least starts.

**Comment by zPaw** [comment]
PR #64 merges this into development. Waiting on a merge into master.
